SDK_CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Releases
 
+### v0.57.0 - Tech Preview (6/4/2019)
+
+* Added new param to control path style access to s3
```

Comment:
May want to mention that this affects the credentials Api

Reply:
Yup. took it out.